### PR TITLE
Updated language files

### PR DESCRIPTION
**Pull request by Somebodyisnobody**
I've updated the language files:
Strings about the registering are now removed, I adjusted the description and made some small precisening improvements.

Please take a look on e6e8670, something happened with stings I didn't touch.

### CODE REVIEW

**Comment by maxmitti** [comment]
Thanks for bringing this up.
I accidentially brought the unused strings back without second thoughts.

Although, your pull request has some issues.
As you have observed, the encoding of umlauts has been changed, probably your editor did it automatically.
Also, IDS_MSG_CMD_HOSTONLY is used for more commands than only /start, so it should either stay the same (or better reword it to something like "Only the host can do that.") or be separated from the other commands.

I think it would be easiest to revert my commit and then merge your improvements, once they are clean enough.

File: planet/System.c4g/LanguageDE.txt [hunk 1]
```diff
@@ -768,7 +768,7 @@ IDS_MSG_CHATDISCLAIMER=Chat - Nutzungsbedingungen
 IDS_MSG_CLEARTHEMESSAGEBOARD=Alle Nachrichten löschen.
 IDS_MSG_CLIENT=Client
 IDS_MSG_CMD_ABORT_NOCOUNTDOWN=Countdown läuft nicht!
-IDS_MSG_CMD_HOSTONLY=Kein Host? Versagt!
+IDS_MSG_CMD_HOSTONLY=Nur der Host kann das Spiel starten!
```
Comment (Fulgen301) [inline planet/System.c4g/LanguageDE.txt]: What's wrong with the old message?

Comment (maxmitti) [reply planet/System.c4g/LanguageDE.txt]: IDS_MSG_CMD_HOSTONLY is used for more commands than only /start, so it should either stay the same (or better reword it to something more kind like "Nur der Host kann das tun.") or be separated from the other commands.

File: planet/System.c4g/LanguageUS.txt [hunk 2]
```diff
@@ -769,7 +769,7 @@ IDS_MSG_CHATDISCLAIMER=Chat - Disclaimer
 IDS_MSG_CLEARTHEMESSAGEBOARD=Clear the message board.
 IDS_MSG_CLIENT=client
 IDS_MSG_CMD_ABORT_NOCOUNTDOWN=Not in countdown!
-IDS_MSG_CMD_HOSTONLY=Host only!
```
Comment (maxmitti) [inline planet/System.c4g/LanguageUS.txt]: IDS_MSG_CMD_HOSTONLY is used for more commands than only /start, so it should either stay the same (or better reword it to something more kind like "Only the host can do that.") or be separated from the other commands.

Comment (maxmitti) [inline planet/System.c4g/LanguageUS.txt]: IDS_MSG_CMD_HOSTONLY is used for more commands than only /start, so it should either stay the same (or better reword it to something more kind like "Only the host can do that.") or be separated from the other commands.

File: planet/System.c4g/LanguageDE.txt (hunk 1 above)
Comment (Somebodyisnobody) [reply planet/System.c4g/LanguageDE.txt]: The old message is nonsense. The english version sounds "Host only!" Which is more precised then this. The old message has a bad slang. So another proposal:
`IDS_MSG_CMD_HOSTONLY=Das kann nur der Host machen.`

**Comment by Somebodyisnobody** [comment]
> As you have observed, the encoding of umlauts has been changed, probably your editor did it automatically.

I used the internal GitHub editor. I don't know how to change that

File: planet/System.c4g/LanguageDE.txt [hunk 3]
```diff
@@ -1,10 +1,10 @@
 IDS_LANG_NAME=Deutsch
-IDS_LANG_INFO=Original-Sprachpaket von RedWolf Design.
```
Comment (maxmitti) [inline planet/System.c4g/LanguageDE.txt]: I'm not sure how we should word this. With your changes it is not only extended, but also modified. Same for english of course.

**Comment by Fulgen301** [comment]
> > As you have observed, the encoding of umlauts has been changed, probably your editor did it automatically.
> 
> I used the internal GitHub editor. I don't know how to change that

Use any text editor with encoding support (Notepad++ for example).

File: planet/System.c4g/LanguageUS.txt (hunk 2 above)
Comment (Somebodyisnobody) [reply planet/System.c4g/LanguageUS.txt]: See 2facd4f852fb3a8ebdb8b2e5f5f747fb2a442ad9

File: planet/System.c4g/LanguageDE.txt (hunk 3 above)
Comment (Somebodyisnobody) [reply planet/System.c4g/LanguageDE.txt]: I also thought about it but there are copyrights on the strings by the authors.

**Comment by Somebodyisnobody** [comment]
After corrosponding with @maxmitti I close this pull request, and apply the proposals to keep the commit-history on the master-branch clean. And I use the online-github editor never again :)
